wp-content/plugins/ig-cms-tree-page-view-extensions/cms-tree-page-view-extensions.php

```diff
@@ -16,3 +16,21 @@ function wpse_73006_submenu_order( $menu_ord )
     return $menu_ord;
 }
 
+
+/**
+ * This function adds a filter ig-cms-tree-view-status that allows attaching
+ * labels to the page tree view.
+ *
+ * @param integer $post_id ID of the post item in the tree view
+ * @param array $post_status contains a list of labels (string)
+ * @return string
+ */
+function ig_tree_view_labels ( $post_id, $post_status ) {
+    $post_status = ( count(array($post_status)) > 0 && $post_status != 'publish' ? array($post_status) : array() );
```

Comment:
The condition `count(array($post_status)) > 0` will always be true, even if `$post_status` is an empty string or `null`.

Reply:
Thanks for capturing that. The array() function does not belong there.

Reply:
Oh yes, strlen() is what I need here.